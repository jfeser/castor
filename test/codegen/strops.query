select([strlen("test") as l, strpos("testing", "in") as p], ascalar(0 as z))