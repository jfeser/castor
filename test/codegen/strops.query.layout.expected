0:1 Scalar (=(Int 0))
